src/pages/terms-and-conditions.js

```diff
@@ -1,85 +1,97 @@
-import { useEffect, useState } from "react"
-import { useRouter } from "next/router"
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
 import { getFirestore, collection, getDocs } from "firebase/firestore";
-import styles from "@/styles/termsAndConditions.module.scss"
-import IndexLayout from "@/Layouts/IndexLayout"
-import Link from "next/link"
-import lt from '@/locales/lt'
-import en from '@/locales/en'
+import styles from "@/styles/termsAndConditions.module.scss";
+import IndexLayout from "@/Layouts/IndexLayout";
+import Link from "next/link";
+import lt from "@/locales/lt";
+import en from "@/locales/en";
 
 export default function TemrsAndConditions() {
-
-   const db = getFirestore();
-   const colRef = collection(db, "ltl-test")
-   getDocs(colRef)
-   .then((snapshot) => {
-      let tncList = []
+  const db = getFirestore();
+  const colRef = collection(db, "ltl-test");
+  getDocs(colRef)
+    .then((snapshot) => {
+      let tncList = [];
       snapshot.docs.forEach((tab) => {
-         tncList.push({ ...tab.data(), id: tab.id })
-      })
-      console.log(tncList)
-   })
-   .catch((err) => {
-      console.log(err.message)
-   })
+        tncList.push({ ...tab.data(), id: tab.id });
+      });
+      console.log(tncList);
+    })
+    .catch((err) => {
+      console.log(err.message);
+    });
 
-   const router = useRouter();
-   const t = router.locale === 'lt' ? lt : en
+  const router = useRouter();
+  const t = router.locale === "lt" ? lt : en;
 
-   const [links, setLinks] = useState([]);
-   const [activeTabLinks, setActiveTabLinks] = useState([]);
+  const [links, setLinks] = useState([]);
+  const [activeTabLinks, setActiveTabLinks] = useState([]);
 
-   useEffect(() => {
-      const getLinks = async () => {
-         const resp = await fetch(`/api/tnc`)
-         const data = await resp.json();
-         setLinks(data)
-         setActiveTabLinks(data[0].links)
-      }
-      getLinks()
-   }, [])
+  useEffect(() => {
+    const getLinks = async () => {
```

Comment:
instead of creating a function you can use IIFE like this: 
    (async () => {
         .... your request code here
    })();

Reply:
very nice ! Done